### DISCUSSION TITLE
Getting started jupyter notebook

### DISCUSSION THREAD

**Discussion by FObermaier** [NetTopologySuite/NetTopologySuite#586] [Show and tell]
I have added a [repository](https://github.com/NetTopologySuite/jupyter) to contain jupyter notebooks that showcase the use of `NetTopologySuite`. As a start, I've converted the [GettingStarted](https://github.com/NetTopologySuite/NetTopologySuite/wiki/GettingStarted) page to a jupyter notebook.

__Please feel encouraged to add your own notebooks!__

**Comment by DGuidi** [comment]
very cool, thanks! 
just a note: I see that some cells (I think "buffer" and "convexhull" only) are marked as `code`, but they should be marked as `markdown`

**Reply by FObermaier** [reply]
Thank you, fixed it.
